Resources/Prototypes/Entities/Objects/Weapons/Throwable/grenades.yml

```diff
@@ -1,10 +1,9 @@
 # TODO probably needs a base grenade
-
 - type: entity
-  name: explosive grenade
-  description: Grenade that creates a small but devastating explosion.
+  name: basic grenade
+  description: How did you get it???
   parent: BaseItem
-  id: ExGrenade
+  id: BasedGrenade
```

Comment:
BaseGrenade, not BasedGrenade. Fix your (intentional, don't do that) typo.

Reply:
```suggestion
  id: BaseGrenade
```